projectDocs/issues/githubIssueTemplateExplanationAndExamples.md

```diff
@@ -10,9 +10,13 @@ If you are not familiar, please take some time to learn about
 **Warning**: In all but exceptional circumstances we require this template to be completed.
 Your issue will likely be closed if this template has not been followed.
 
-We currently have two templates, both are described on this page:
+We currently have four templates:
 - for bug reports
-- for feature requests.
+- for feature requests
+- for special case issues that cannot be easily categorised as either a bug report or a feature request
+- for security vulnerabilities

```

Comment:
I would avoid conflating issues with advisories as they are reported very differently. I would at the very least note the difference and link to this: https://github.com/nvaccess/nvda/blob/master/security.md